Spell check (#65)
rapidez/docs: src/3.x/configuration.md

## Magento configuration

> Some Magento configuration options involve the frontend, as the Magento frontend is not used, most of them don't do anything. Rapidez listens to some of those configurations:

Reviewer: ```suggestion
Some Magento configuration options involve the frontend. As the Magento frontend is not used, most of them don't do anything. Rapidez listens to some of those configurations:
```

`design/search_engine_robots/default_robots` | Meta robots tag value
`design/search_engine_robots/custom_instructions` | See [Robots.txt](configuration.html#robots-txt)

If you need to access a Magento configuration, you can use the [`@config` Blade Directive](theming.html#config) or the Rapidez facade `Rapidez::config()`, which accepts the same parameters as the directive. For more advanced usage (like defining which scope), you can use:
```php
\Rapidez\Core\Models\Config::getValue($path, ConfigScopes::SCOPE_STORE, $scopeId, $options)
```

## Base URL

> Rapidez will be the frontend, Magento just as the backend. So they'll need two different URLs; we suggest setting the `web/secure/base_url` differently per scope:

Reviewer: ```suggestion
Rapidez will become the frontend, Magento will only be used for the backend. Because of this, they'll need two different URLs. We suggest setting the `web/secure/base_url` differently per scope:
```

## Early hints

> We come with [Early Hints](https://github.com/justbetter/laravel-http3earlyhints) built-in. You should enable your early hints in [Cloudflare](https://developers.cloudflare.com/cache/advanced-configuration/early-hints/#enable-early-hints) or your CDN for a boost. Anything loaded with `@vite()` will properly get loaded with early hints, it will also discover other elements on your page which could benefit from early hints. Like your `preload` and `preconnect` links, eager-loaded images, blocking scripts, and blocking styles. You can disable it with:

Reviewer: ```suggestion
Rapidez comes with [Early Hints](https://github.com/justbetter/laravel-http3earlyhints) built in. You should enable your early hints in [Cloudflare](https://developers.cloudflare.com/cache/advanced-configuration/early-hints/#enable-early-hints) or your CDN for a boost. Anything loaded with `@vite()` will properly get loaded with early hints, it will also discover other elements on your page which could benefit from early hints. Like your `preload` and `preconnect` links, eager-loaded images, blocking scripts, and blocking styles. You can disable it with:
```
